release.Dockerfile

```
@@ -2,6 +2,7 @@ FROM python:3.9-slim
 
 # Environment Variables
 ENV ARGILLA_HOME_PATH=/var/lib/argilla
+ENV ADMIN_ENABLED=true
```

Since this variable is related to the default user creation, maybe we can rename it to `DEFAULT_USER_ENABLED`?
```suggestion
ENV DEFAULT_USER_ENABLED=true
```